README.md

```diff
@@ -14,25 +14,40 @@ Clone the repository and install dependencies:
 git clone https://github.com/ambanum/CGUs.git
 cd CGUs
 npm install
-# Git clone CGUs data
 ```
 
 ## Setting up the database
 
-You can create a new database to start your own history from now:
-
+Initialize the database:
 ```sh
-cd ..
-mkdir cgus-data
-cd cgus-data
-git init
+npm run setup
 ```
 
-Or you can download the entire history of terms of services:
+## Configuration
 
-```sh
-cd ..
-git clone https://github.com/ambanum/cgus-data/
+Some configs are available and can be changed in `config/default.json`:
+
+```json
+{
+  "serviceProvidersPath": "Directory containing providers definition and associated sanitizers.",
+  "history": {
+    "dataPath": "Database directory path relative to the root of this project",
+    "configureAuthor": "Boolean. Set to true to automatically configure the git author. Used in production and CI envs.",
```

Comment:
👍

Reply:
Not really because we always want author to be defined in config (even on your local machine, you want AmbNum to be the commiter), but on fresh env (like CI or production), git has no global user defined and it needs one.

Reply:
Apparently we can avoid setting the global user this way: https://stackoverflow.com/questions/22058041/commit-without-setting-user-email-and-user-name 🙂